src/main/java/picard/analysis/replicates/CollectIndependentReplicateMetrics.java

```diff
@@ -102,50 +102,50 @@
  */
 
 @CommandLineProgramProperties(
-        usage = "Estimates the rate of independent replication rate of reads within a bam. \n" +
+        summary = "Estimates the rate of independent replication rate of reads within a bam. \n" +
                 "That is, it estimates the fraction of the reads which would be marked as duplicates but " +
                 "are actually biological replicates, independent observations of the data. ",
-        usageShort = "Estimates the rate of independent replication of reads within a bam.",
+        oneLineSummary = "Estimates the rate of independent replication of reads within a bam.",
         programGroup = Alpha.class
 )
 public class CollectIndependentReplicateMetrics extends CommandLineProgram {
     private static final int DOUBLETON_SIZE = 2, TRIPLETON_SIZE = 3;
```

Comment:
I saw this line and said to myself, "this has to be a @yfarjoun class", and sure enough... 😉